blueocean-dashboard/src/main/js/util/UrlUtils.js

```diff
@@ -19,6 +19,14 @@ export const buildPipelineUrl = (organization, fullName, tabName) => {
     return tabName ? `${baseUrl}/${tabName}` : baseUrl;
 };
 
+export const buildClassicConfigUrl = (window, pipeline) => {
+    if (pipeline && pipeline.fullDisplayName) {
+        return `${window.jenkinsCIGlobal.rootURL}/job/${pipeline.fullDisplayName.split('/').join('/job/')}/configure`;
```

Comment:
like yes to all the above - thanks!

Is there an example of UrlConfig? (I can probably find it). window felt bad too... but I am a dirty hacker. 

@cliffmeyers good point - yes will change it to full name (I will check it works)